### PR TITLE
i18n(es): Translate incorrect-strategy-for-i18n.mdx

### PR DESCRIPTION
**Pull request by helmerdavila**
#### Description (required)

- Translating `incorrect-strategy-for-i18n` to spanish

### CODE REVIEW

File: src/content/docs/es/reference/errors/incorrect-strategy-for-i18n.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: You can't use the current function with the current strategy
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **IncorrectStrategyForI18n**: La función `NOMBRE_DE_FUNCION\' sólo puede ser usada cuando `i18n.routing.strategy` está asignado como `"manual"`.
```
Comment (liruifengv) [inline src/content/docs/es/reference/errors/incorrect-strategy-for-i18n.mdx:7]:
```suggestion
> **IncorrectStrategyForI18n**: La función `NOMBRE_DE_FUNCION` sólo puede ser usada cuando `i18n.routing.strategy` está asignado como `"manual"`.
```

fixed this in https://github.com/withastro/docs/pull/7876

File: src/content/docs/es/reference/errors/incorrect-strategy-for-i18n.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## ¿Qué salió mal?
+Algunas funciones de internacionalización sólo están disponibles cuando el ruteado de Astro está deshabilitado por la configuración `i18n.routing: "manual"`.
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/incorrect-strategy-for-i18n.mdx:10]:
```suggestion
Algunas funciones de internacionalización sólo están disponibles cuando el enrutamiento de Astro está deshabilitado por la configuración `i18n.routing: "manual"`.
```

File: src/content/docs/es/reference/errors/incorrect-strategy-for-i18n.mdx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+**Ver también:**
+-  [Ruteado con `i18n`](/es/guides/internationalization/#routing)
```
Comment (Waxer59) [inline src/content/docs/es/reference/errors/incorrect-strategy-for-i18n.mdx:13]:
```suggestion
-  [Enrutamiento con `i18n`](/es/guides/internationalization/#routing)
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by helmerdavila** [comment]
@liruifengv and @Waxer59 
Everything fixed

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**